packages/backend/README.md

```diff
@@ -48,7 +48,7 @@ This package provides Clerk Backend API resources and low-level authentication u
 
 ## How to use
 
-Works on Node >= 16 or on any V8 Isolates runtimes such as Cloudflare Workers.
+Works on Node.js `^18.18` (or later) or on any V8 Isolates runtimes (eg Cloudflare Workers).
```

Comment:
In the README you write `^18.18` but in the `engines` field we have `>=18`.

Currently that would mean we allow `^18.0.0`. So if you want to have `^18.18` you'll need to adjust the `engines` field everywhere.